Cargo.toml

```diff
@@ -376,10 +376,11 @@ lto = "fat"
 codegen-units = 1
 
 # A much faster to compile version of `release`.
```

Comment:
```suggestion
# A much faster to compile version of `release` for development use.
```

Reply:
Thanks! I had grepped in the docs, but didn’t think of the cargo.toml